### PR TITLE
Test on old and new versions of fsharpc

### PR DESCRIPTION
**Pull request by mavnn**

### CODE REVIEW

**Comment by sergey-tihon** [comment]
@forki Also interesting that `Paket` seems fail on mono `3.2.8`
>Downloading Nuget.CommandLine 3.3.0
Could not fix timestamps in /home/travis/build/fsprojects/FSharp.TypeProviders.StarterPack/packages/Nunit.Runners/Nunit.Runners.2.6.4.nupkg. Error: The requested feature is not implemented.
Paket failed with:
	Error during extraction of /home/travis/build/fsprojects/FSharp.TypeProviders.StarterPack/packages/Nunit.Runners/Nunit.Runners.2.6.4.nupkg.
Message: The requested feature is not implemented.

**Comment by forki** [comment]
Yes older mono has issues with zip. Upgrade your mono.

**Comment by sergey-tihon** [comment]
The goal on this PR is to test that type provides works on different versions of Mono
```
mono:
 +  - latest
 +  - 3.2.8
```

**Comment by forki** [comment]
Yes but I can't magically fix older mono.

**Comment by mavnn** [comment]
How important do we feel keeping support for older versions of F# is for this repo?

**Comment by mavnn** [comment]
@dsyme - an opinion on how backwards ProvidedTypes should be for older F# versions? At the moment we get a warning for Seq.item, but if we change it to Seq.nth than it will fail building with older versions.

**Comment by mavnn** [comment]
Those should be the other way around, obviously. I'll go drink some :coffee:

File: .travis.yml [hunk 1]
```diff
@@ -1,4 +1,7 @@
 language: csharp
+mono:
+  - latest
+  - 3.2.8
```
Comment (dsyme) [inline .travis.yml]: 3.2.8 is very very old, we generally require 4.0.5 or above

**Comment by dsyme** [comment]
Closing in favour of #79

**Pull request closed without merging**